Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';

```

Comment:
@RickyRapp Why do you have a react code here? this is a repo of server || client?